diff --git a/atuin-client/src/database.rs b/atuin-client/src/database.rs
index e67fe06d..81db9104 100644
--- a/atuin-client/src/database.rs
+++ b/atuin-client/src/database.rs
@@ -139,9 +139,9 @@ impl Sqlite {
         }
 
         let opts = SqliteConnectOptions::from_str(path.as_os_str().to_str().unwrap())?
-            .journal_mode(SqliteJournalMode::Wal)
+            .journal_mode(SqliteJournalMode::Memory)
             .optimize_on_close(true, None)
-            .synchronous(SqliteSynchronous::Normal)
+            .synchronous(sqlx::sqlite::SqliteSynchronous::Off)
             .create_if_missing(true);
 
         let pool = SqlitePoolOptions::new()
diff --git a/atuin-client/src/record/sqlite_store.rs b/atuin-client/src/record/sqlite_store.rs
index 6333bb27..17fa82b9 100644
--- a/atuin-client/src/record/sqlite_store.rs
+++ b/atuin-client/src/record/sqlite_store.rs
@@ -41,7 +41,8 @@ impl SqliteStore {
         }
 
         let opts = SqliteConnectOptions::from_str(path.as_os_str().to_str().unwrap())?
-            .journal_mode(SqliteJournalMode::Wal)
+            .journal_mode(SqliteJournalMode::Memory)
+            .synchronous(sqlx::sqlite::SqliteSynchronous::Off)
             .foreign_keys(true)
             .create_if_missing(true);
 
