diff --git a/src/libutil/compression.cc b/src/libutil/compression.cc
index d06f1f87b..7bed3e4dc 100644
--- a/src/libutil/compression.cc
+++ b/src/libutil/compression.cc
@@ -48,7 +48,7 @@ struct ArchiveDecompressionSource : Source
             archive = std::make_unique<TarArchive>(src, true);
             this->archive->check(archive_read_next_header(this->archive->archive, &ae),
                 "failed to read header (%s)");
-            if (archive_filter_count(this->archive->archive) < 2) {
+            if (archive_filter_count(this->archive->archive) < 1) {
                 throw CompressionError("input compression not recognized");
             }
         }
